code/game/objects/items/sticker.dm

```diff
@@ -0,0 +1,116 @@
+/obj/item/sticker
+	name = "sticker"
+	desc = "A sticker with some strong adhesive on the back, sticks to stuff!"
+	item_flags = NOBLUDGEON | XENOMORPH_HOLDABLE //funny
+	resistance_flags = FLAMMABLE
+	icon = 'icons/obj/stickers.dmi'
+	w_class = WEIGHT_CLASS_TINY
+	throw_range = 3
+	vis_flags = VIS_INHERIT_DIR | VIS_INHERIT_PLANE | VIS_INHERIT_LAYER
+	var/mutable_appearance/sticker_overlay
+	var/list/icon_states
+	var/atom/attached
+
+/obj/item/sticker/Initialize(mapload)
+	. = ..()
+	if(icon_states)
```

Comment:
We have a component which handles this for mobs, it might not be a terrible idea to have it for items too... I won't require it for this PR though.